cmd/describe/cluster/cmd.go

```diff
@@ -41,6 +41,9 @@ const (
 
 	EnabledOutput  = "Enabled"
 	DisabledOutput = "Disabled"
+
+	YesOutput = "Yes"
```

Comment:
@ciaranRoche We have constants for these already in `pkg/output/print.go`, suggest we use those instead.